epigraphhub/settings.py

```diff
@@ -0,0 +1,60 @@
+from typing import List
+
+from dataclasses import dataclass, field
```

Comment:
I think this first configuration approach based on `dataclasses` can be replaced entirely by the `.env` based configuration.

Reply:
as I mentioned before, .env files is good for devops, but it is not a common approach for libraries.